iwyu.cc

```diff
@@ -1936,8 +1934,9 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
                                    "(for autocast)");
         }
       } else {
-        VERRS(6) << "WARNING: NULL TypeSourceInfo for " << PrintableDecl(*param)
-                 << " (type " << PrintableType(param_type) << ")\n";
+        VERRS(6) << "WARNING: nullptr TypeSourceInfo for "
+                 << PrintableDecl(*param) << " (type "
+                 << PrintableType(param_type) << ")\n";
```

Comment:
Not critical, but I think it might be slightly more readable to have in-text parenthesis on the same line like
```c++
        VERRS(6) << "WARNING: nullptr TypeSourceInfo"
                 << " for " << PrintableDecl(*param)
                 << " (type " << PrintableType(param_type) << ")\n";
```